packages/overlays/src/OverlayMixin.js

```diff
@@ -3,7 +3,7 @@ import { OverlayController } from './OverlayController.js';
 
 /**
  * @type {Function()}
- * @polymerMixin
+ * @polymerMixinOverlayMixin
```

Comment:
Not sure which static analysis tool looks for `@polymerMixin` and if we still use that (do we need it for webcomponent-analyzer @daKmoR?), but I think we need and extra space here :)